nf_core/pipeline-template/subworkflows/nf-core/utils_nfcore_pipeline/main.nf

```diff
@@ -107,15 +107,31 @@ def processVersionsFromYAML(yaml_file) {
     return yaml.dumpAsMap(versions).trim()
 }
 
+//
+// Get topic version for make it a YAML
+// Expects: tuple val("$task.process"), val("PINTS"), eval("pints_caller --version")
+//
+def topicVersionToYAML(topic_version) {
+    def (taskProcess, tools, versions) = topic_version
+
+    def toolsVersions = [tools, versions]
+        .transpose()
+        .collect { k, v -> "${k}: ${v}" }
+    return """
+    |${taskProcess.tokenize(':').last()}:
+    |  ${toolsVersions.join('\n|  ')}
+    """.stripMargin().trim()
+}
+
 //
 // Get workflow version for pipeline
 //
 def workflowVersionToYAML() {
     return """
-    Workflow:
-        $workflow.manifest.name: ${getWorkflowVersion()}
-        Nextflow: $workflow.nextflow.version
-    """.stripIndent().trim()
+    |Workflow:
+    |  $workflow.manifest.name: ${getWorkflowVersion()}
+    |  Nextflow: $workflow.nextflow.version
```

Comment:
what does this syntax do here?

Reply:
It strips the white space from the multi line string until the delimiter which by default is |

Reply:
ah nice :)